tests/shared/src/test/scala/scala/meta/tests/parsers/dotty/NamedTuplesSuite.scala

```diff
@@ -0,0 +1,96 @@
+package scala.meta.tests.parsers.dotty
+
+import scala.meta._
+
+class NamedTuplesSuite extends BaseDottySuite {
+
+  test("simple-named-term") {
+    runTestAssert[Stat]("(a = 123, b = \"123\")")(Term.Tuple(
+      List(Term.Assign(Term.Name("a"), Lit.Int(123)), Term.Assign(Term.Name("b"), Lit.String("123")))
+    ))
+  }
+
+  test("complex-named-term") {
+    runTestAssert[Stat]("(a = 123, b = \"123\", c = (a = 123, d = 2.01d))")(Term.Tuple(List(
+      Term.Assign(Term.Name("a"), Lit.Int(123)),
+      Term.Assign(Term.Name("b"), Lit.String("123")),
+      Term.Assign(
+        Term.Name("c"),
+        Term.Tuple(List(
+          Term.Assign(Term.Name("a"), Lit.Int(123)),
+          Term.Assign(Term.Name("d"), Lit.Double(2.01d))
+        ))
+      )
+    )))
+  }
+
+  test("simple-named-type") {
+    runTestAssert[Type]("(a: Int, b: String)")(Type.Tuple(List(
```

Comment:
is this from the same sip, or in the same vein?

Reply:
It's the same sip, it's the type of the named tuples, but it already works, so decided to not add a new AST node.

Reply:
of course, don't need new trees